### PR TITLE
Made plugin compatible with Python 3 (while keeping Python 2 compat)

### PR DESCRIPTION
**Pull request by habbbe**

### CODE REVIEW

File: japanese/lookup.py [hunk 1]
```diff
@@ -102,7 +119,10 @@ def _field(name):
     try:
         return mw.reviewer.card.note()[name]
     except:
-        return
+        try:
+            return mw.reviewer.card.note().items()[name][1]
+        except:
+            return
```
Comment (dae) [inline japanese/lookup.py]: What is the purpose of this change?

File: japanese/lookup.py [hunk 2]
```diff
@@ -5,22 +5,39 @@
...
+    setUrl = QUrl.setUrl # Python 3+
+
 class Lookup(object):
 
     def __init__(self):
         pass
 
     def selection(self, function):
         "Get the selected text and look it up with FUNCTION."
+        #TODO This doesn't seem to work...
```
Comment (dae) [inline japanese/lookup.py]: The action is executed asynchronously, so you need to either wait a while or grab the clipboard in javascript via evalWithCallback()

**Comment by dae** [comment]
thanks, I've made some comments

File: japanese/lookup.py (hunk 1 above)
Comment (habbbe) [inline japanese/lookup.py]: The new way to connect actions discards the default argument of onLookupExpression. I will fix this by connecting a lambda to the "...expression on alc" action instead (and roll back this change).

File: japanese/lookup.py (hunk 2 above)
Comment (habbbe) [inline japanese/lookup.py]: Would you prefer to keep it asynchronous/lazy, or is the solution I provided sufficient?

Comment (habbbe) [inline japanese/lookup.py]: On the same note: is it good practice to replace the clipboard like this without telling the user? I'm also interested in why you decided to use the clipboard for this. I'm sure you had a good reason. :)

Comment (dae) [inline japanese/lookup.py]: I don't remember why it's like that - it's possible it just wasn't clear at the time how to get the selection otherwise.

Does the solution you provided work on both 2.0 and 2.1? I thought by your comment it may not be working, but perhaps you were just referring to the commented text? If it's just that the commented text doesn't work, feel free to snip it out completely.

File: japanese/lookup.py (hunk 1 above)
Comment (dae) [inline japanese/lookup.py]: Sounds good, thanks.

**Comment by habbbe** [comment]
I have updated the pull request. Regarding the copy: It works on both branches (I tried on 2.1.0 and 2.0.45), so I removed the older code.

**Comment by dae** [comment]
thank you for contributing!
